design-documents/graph-ql/coverage/add-items-to-cart-single-mutation.md

```diff
@@ -0,0 +1,205 @@
+# Single mutation for adding products to cart
+In order to simplify the flow of adding different type of products to wishlist and cart it is proposed to use single mutation for all product types.
+
+*Proposed schema for adding products to cart* - [AddProductsToCart](AddProductsToCart.graphqls)
+
+*Proposed schema for adding products to wish list* - [AddProductsToWishlist](AddProductsToWishlist.graphqls)
+
+## Definition
+
+- Bundle dynamic, grouped and configurable product are just templates which help merchant to select a set of simple products.
```

Comment:
Why do we need this? It is not completely accurate statement.

Reply:
this is desired state